src/core/proto/route_table_mgr.cpp

```diff
@@ -152,113 +152,117 @@ void route_table_mgr::update_tbl(nl_data_t data_type)
     netlink_socket_mgr::update_tbl(data_type);
 }
 
-void route_table_mgr::parse_entry(struct nlmsghdr *nl_header)
+void route_table_mgr::parse_entry(struct nl_object *nl_obj)
```

Comment:
add netlink/route/route.h and netlink/netlink.h includes.
Currently the route.h is included implicitly from netlink_event.h which can be rewritten in the future to generalize the netlink related code. Besides, we can do forward declarations for few netlink structures and avoid netlink in the XLIO headers.

Reply:
done :)